packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -1065,9 +1080,18 @@ class TextStyle with Diagnosticable {
     );
   }
 
-  /// Interpolate between two text styles.
+  /// Interpolate between two text styles for animated transitions.
```

Comment:
Why include the "for animation transitions" qualifier?

Reply:
The implementation does things to keep the transition smooth (e.g.: `fontSize: ui.lerpDouble(a.fontSize ?? b.fontSize, b.fontSize ?? a.fontSize, t),`), that doesn't really make sense for anything other than transitions.